pkg/storage/tree/flamebearer.go

```diff
@@ -19,6 +21,8 @@ type Flamebearer struct {
 	Format     Format `json:"format"`
 }
 
+var lostDuringRenderingName = jsonableSlice("other")
```

Comment:
Maybe we could change this to something else in the future, e.g "lost during rendering"